pkgs/top-level/all-packages.nix

```
@@ -15620,6 +15620,8 @@ with pkgs;
     jdk = jdk8; # TODO: remove override https://github.com/NixOS/nixpkgs/pull/89731
   };
 
+  jinja2-cli = callPackage ../development/tools/jinja2-cli { };
```

```suggestion
  jinja2-cli = callPackage ../development/tools/jinja2-cli {
    inherit (python3Packages) python3Packages fetchPypi jinja2 pyyaml toml xmltodict;
  };
```
(Also change the parts that use `python3Packages`)

---

This isn't really problematic for Python applications in the top level though.